test/fixtures/gateway_account_fixtures.js

```diff
@@ -45,6 +45,38 @@ function validGatewayAccount (opts) {
 }
 
 module.exports = {
+  validGatewayAccountGooglePayToggleRequest: (enabled = false) => {
```

Comment:
Since these two methods are incredibly similar I'm wondering if this could instead be a `validGatewayAccountPatchRequest(op, path, value)` so it can be used for any patch request? You can keep the understanding of what the request is when its used e.g. 
```
const validToggleApplePayPatchRequest = validGatewayAccountPatchRequest(......
```

Reply:
👍 Updated. Thanks

Reply:
I've renamed this so that it's a "generic wallet payment toggle" request: `validGatewayAccountWalletToggleRequest`